ibis/pyspark/compiler.py

```diff
@@ -1052,32 +1074,39 @@ def compile_window_op(t, expr, scope, timecontext, **kwargs):
     ]
 
     order_by = window._order_by
+    # Timestamp needs to be cast to long for filtering in spark
     ordering_keys = [
-        key.to_expr().get_name()
-        for key in map(operator.methodcaller('op'), order_by)
+        F.col(expr.get_name()).cast('long')
+        if isinstance(expr.op().args[0], types.TimestampColumn)
```

Comment:
What is `expr.op().args[0]` ? 

Ideally we should not use position index into args, we should the actual arg variable instead.

Reply:
```
(Pdb) type(order_by)
<class 'list'>
(Pdb) type(order_by[0])
<class 'ibis.expr.types.SortExpr'>
(Pdb) type(order_by[0].op())
<class 'ibis.expr.operations.SortKey'>
(Pdb) type(order_by[0].op().args)
<class 'tuple'>
(Pdb) type(order_by[0].op().args[0])
<class 'ibis.expr.types.TimestampColumn'>
```
We will need to get to the column in root table. Could you suggest better ways to get to the type for column?

Reply:
I updated it, rename `expr` to be `sort_expr` and `sort_expr.op().expr` is the column we need here.